### DISCUSSION TITLE
Deactivate menu

### DISCUSSION THREAD

**Discussion by AniGerm** [arendst/Tasmota#20347] [Support request]
Hello,
is it possible to deactivate the configuration menu? Maybe some console command? I would like to set the device up and “secure” it against some other users of the Network. Thank you!

**Comment by sfromis** [comment]
As long as you do not take "secure" as hacking-proof, you can have it run in user mode instead of admin, where most web UI elements are gone and deactivated. Or you can disable the web server completely. Use MQTT if you want to be able to revert.
https://tasmota.github.io/docs/Commands/#webserver

https://tasmota.github.io/docs/Commands/#webpassword also exists. Very rudimentary, and and no "real security", but does take a bit of effort to "penetrate".
